src/ui/Icon.js

```diff
@@ -80,6 +82,7 @@ const IconsMap = {
   [Icons.TRASH]: faTrash,
   [Icons.BINOCULARS]: faBinoculars,
   [Icons.EXCLAMATION_TRIANGLE]: faExclamationTriangle,
+  [Icons.PENCIL_ALT]: faPencilAlt,
```

Comment:
Nit: I'm wondering if we should really keep the `_ALT` in the name? On the one hand it makes it predictable as long as we keep using FontAwesome, but if we ever switch to another icon set later it might not make sense anymore 🤔 And it would be cool if we could just switch the icons here then and not have to search & replace all usages. Because then it kind of defeats the purpose of abstracting it away in our own icon component.

Reply:
If we change it we might also want to change it for `SIGN_OUT_ALT`

Reply:
Yes for now we've mapped it 1 to 1 with fontawesome naming. I think we're not going to change away from fontawesome any time soon, or are we? 😄

Reply:
No, not immediately, but I would keep open the possibility at least. The goal of our UI components was to abstract away the implementation, like we also do with the Bootstrap components by not exposing all props etc, and using our own names. So I was thinking we should do the same for the icons?

Reply:
@bertramakers Maybe create a ticket for this and we'll pick that up later?

Reply:
Why? It's only added in this PR so might as well change it right now no?

Reply:
oke done